src/Uno.UI/UI/Xaml/Media/Brush.Android.cs

```diff
@@ -32,26 +32,38 @@ public partial class Brush
 		/// </summary>
 		/// <param name="destinationRect">RectF that will be drawn into - used by ImageBrush</param>
 		/// <returns>A Paint with Fill style</returns>
-		internal Paint GetFillPaint(Windows.Foundation.Rect destinationRect)
+		internal void ApplyToFillPaint(Windows.Foundation.Rect destinationRect, Paint paint)
 		{
-			var paint = GetPaintInner(destinationRect);
-			paint?.SetStyle(SystemFill);
-			return paint;
+			if (paint is null)
+			{
+				throw new ArgumentNullException(nameof(paint));
+			}
+
+			paint.Reset();
+			paint.AntiAlias = true;
+			ApplyToPaintInner(destinationRect, paint);
+			paint.SetStyle(SystemFill);
 		}
 
 		/// <summary>
 		/// Return a paint with Stroke style
 		/// </summary>
 		/// <param name="destinationRect">RectF that will be drawn into - used by ImageBrush</param>
 		/// <returns>A Paint with Stroke style</returns>
-		internal Paint GetStrokePaint(Windows.Foundation.Rect destinationRect)
+		internal void ApplyToStrokePaint(Windows.Foundation.Rect destinationRect, Paint paint)
 		{
-			var paint = GetPaintInner(destinationRect);
-			paint?.SetStyle(SystemStroke);
-			return paint;
+			if (paint is null)
+			{
+				throw new ArgumentNullException(nameof(paint));
+			}
+
+			paint.Reset();
+			paint.AntiAlias = true;
+			ApplyToPaintInner(destinationRect, paint);
+			paint.SetStyle(SystemStroke);
```

Comment:
Can the SetStyle be done before ApplyToPaintInner? If so we may be able to bundle reset, antialias and SetStyle in one native call to java.